vignettes/estimate_infections_workflow.Rmd

```diff
@@ -258,14 +290,15 @@ All the options are combined in a call to the `estimate_infections()` function.
 For example, using some of the options described above one could call
 
 
-```r
+``` r
 def <- estimate_infections(
   example_confirmed, 
   generation_time = gt_opts(generation_time),
   delays = delay_opts(delay),
   rt = rt_opts(prior = rt_prior)
 )
-#> Warning: There were 3 divergent transitions after warmup. See
+#> DEBUG [2024-09-19 14:46:51] estimate_infections: Running in exact mode for 2000 samples (across 4 chains each with a warm up of 250 iterations each) and 147 time steps of which 7 are a forecast
+#> Warning: There were 1 divergent transitions after warmup. See
```

Comment:
I'm slightly confused that the number of divergent transitions has been reduced (and the results plot updated) given that we're setting a seed - which `rstan` version are you using to render these? I suspect this might could be a similar version discrepancy to what makes the actions fail (see #769) so we should make sure we're rendering everything with the same `rstan` version (the one from the stan dev universe).

Reply:
My current version is Version: 2.32.6. I'll update now.

Reply:
I'm not sure why. I think we had this same seed conversation sometime back. Let me pull up the conversation.

Reply:
Here: https://github.com/epiforecasts/EpiNow2/issues/705

Reply:
> Here: #705

This was traced down to differences in rstan version (2.32.6 vs. 2.35.0.9000) with an update to the random generator.

Reply:
have these vignettes been updated since i refactored the models? I think the answer is no and we are seeing good results from that effort here

Reply:
Yeah, I'm also realising much fewer divergences in the [work on forecasting with temporally aggregated data](https://github.com/tatendashoko/now-fore-cast) after a few tweaks in the sampler.